app/components/target-audience-select.hbs

```diff
@@ -0,0 +1,11 @@
+<AuLabel class="au-u-margin-top" for="filter-municipality">Doelgroep</AuLabel>
+<AuCheckboxGroup
+  @alignment="default"
+  @selected={{@selected}}
+  @onChange={{@onChange}}
+  as |Group|
+>
+  <Group.Checkbox @value="-18">-18</Group.Checkbox>
+  <Group.Checkbox @value="18+">18+</Group.Checkbox>
+  <Group.Checkbox @value="65+">65+</Group.Checkbox>
```

Comment:
Is there a plan to define and use a codelist for this at some point ?

Reply:
@claire-lovisa I don't think we will use a codelist for this.